docs/pages/checkbox.js

```diff
@@ -757,6 +757,88 @@ function CheckboxExample() {
     </Fieldset>
   );
 }
+`}
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection
+          title="Label visibility"
```

Comment:
Small nit to add a line to the "labels" section in Accessibility about this example